cpp/include/raft/matrix/detail/select_radix.cuh

```diff
@@ -1026,7 +1026,7 @@ _RAFT_DEVICE void filter_and_histogram_for_one_block(const T* in_buf,
       const auto previous_bits = (twiddle_in(value, select_min) >> previous_start_bit)
                                  << previous_start_bit;
       if (previous_bits == kth_value_bits) {
-#if CUDART_VERSION < 12000
+#if CUDART_VERSION < 12000 && __CUDA_ARCH__ < 890
```

Comment:
Hi @yong-wang , could you take a moment to review it? Many thanks!

Reply:
This change will prevent the `volatile` applied on H100.

Reply:
Should be `< 900`?